src/components/pages/BookPage/bookPage.js

```diff
@@ -80,7 +77,7 @@ const BookPage = () => {
               </div>
               <div className="mt-4">
                 <MDBBtn color="primary" className="px-4 rounded-pill">
-                  {bookPageTranslate('addToCart')}
+                  {bookPageTranslate('bookPage.addToCart')}
```

Comment:
can you extract all the translation one time as an object and then use it every where
```
const bookPageText = bookPageTranslation('bookPage', { returnObjects: true })
```
this code will return the whole obj and you can use it